seekret-rules/aws.rule

```diff
@@ -1,8 +1,12 @@
 secret_access_key:
   match: (\"|')?(AWS|aws|Aws)?_?(SECRET|secret|Secret)?_?(ACCESS|access|Access)?_?(KEY|key|Key)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9/\+=]{40}(\"|')?
+  unmatch:
+    - (\"|')?(NEW|New|new)_?(RELIC|Relic|relic)_?(KEY|key|Key)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9/\+=]{40}(\"|')?
 
 access_key_id:
   match: (\"|')?(AWS|aws|Aws)_?(ACCESS|access|Access)?_?(KEY|key|Key)_?(ID|id|Id)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9]{20}(\"|')?
+  unmatch:
+    - (\"|')?(NEW|New|new)_?(RELIC|Relic|relic)_?(KEY|key|Key)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9]{20}(\"|')?
```

Comment:
Pretty sure this is doing more than it has to.. I should only have to find some combination of "new" and "relic".